src/Search/Console.php

```diff
@@ -54,6 +58,28 @@
         ],
     ];
 
+    public function queuePushCommand(InputInterface $input, OutputInterface $output)
+    {
+        $helper = new QuestionHelper();
+        // Get the type.
+        $choice = new ChoiceQuestion('<question>Which type would you like to import</question>', [
+            'research-article',
+            'blog-articles',
+            'collection',
+        ], 0);
+        $type = $helper->ask($input, $output, $choice);
+        // Ge the Id.
+        $choice = new Question('<question>Whats the ID of the item to import: </question>');
```

Comment:
being asked is useful because typically one has no idea about the possible options. It would be useful to also have a way of running this as a single command line, you can imagine having to run this many times and that will be faster to repeat a test